dashboard/app/helpers/aichat_openai_helper.rb

```diff
@@ -0,0 +1,48 @@
+module AichatOpenaiHelper
```

Comment:
nit: mind adding some comments on what this helper is for (especially to distinguish from `OpenAiChatHelper`)?

Reply:
Good call, will do!